jenkins-simulate-data.sh

```diff
@@ -62,8 +64,16 @@ fi
 # use "submitted_unaligned_reads" if it's there ...
 #
 leafNode="submitted_unaligned_reads"
-if ! jq -r '.|values|map(select(.category=="data_file"))|map(.id)|join("\n")' < "$TEST_DATA_PATH/schema.json" | grep "$leafNode" > /dev/null; then
-  leafNode="$(jq -r '.|values|map(select(.category=="data_file"))|map(.id)|join("\n")' < "$TEST_DATA_PATH/schema.json" | head -1)"
+
+# diff testedEnvs require diff nodes of type "file"
+if [ "$testedEnv" == "data.midrc.org" ] || [ "$testedEnv" == "qa-midrc.planx-pla.net" ]; then
+  if ! jq -r '.|values|map(select(.category=="imaging_data_file"))|map(.id)|join("\n")' < "$TEST_DATA_PATH/schema.json" | grep "$leafNode" > /dev/null; then
```

Comment:
why not select the first `_file` node instead?

Reply:
if there's a way to replace `map(select(.category=="data_file"))` with `*file`

Reply:
We can take this one for a spin and improve it later. 🤷‍♂️ Whatever prevents any force-merges and unblocks midrc work.